expression/builtin/builtin.go

```diff
@@ -69,7 +69,9 @@ var Funcs = map[string]Func{
 	// time functions
 	"curdate":           {builtinCurrentDate, 0, 0, false, false},
 	"current_date":      {builtinCurrentDate, 0, 0, false, false},
+	"current_time":      {builtinCurrentTime, 0, 1, false, false},
 	"current_timestamp": {builtinNow, 0, 1, false, false},
+	"curtime":           {builtinCurrentTime, 0, 1, false, false},
 	"date":              {builtinDate, 8, 8, true, false},
```

Comment:
@siddontang Why `builtinDate`'s `minArgs` and `maxArgs` are both set to `8`? `DATE` should accept exactly one argument in the MySQL doc (`DATE(expr)`).
BTW, previously when I set `current_time` and `curtime`'s `maxArgs` to `1`, it can still pass all the tests. This makes me doubt whether this kind of checks really work.
Could you help me explain these? Thanks!